curvefs/src/common/s3util.h

```diff
@@ -32,11 +32,16 @@ namespace s3util {
 inline std::string GenObjName(uint64_t chunkid, uint64_t index,
                               uint64_t compaction, uint64_t fsid,
                               uint64_t inodeid) {
-    return std::to_string(fsid) + "_" + std::to_string(inodeid) + "_" +
+    return std::to_string(fsid) + "/" +
+           std::to_string(inodeid/1000/1000) + "/" +
```

Comment:
Compatibility with existing systems needs to be considered.

Reply:
There is no better way to handle with compatibility. If changing the data organization, we have to sync the existing file system to new file system.